### ISSUE TITLE
Feature request: forwarder activity log

### ISSUE THREAD

**Issue by furanzoni** [tallbl0nde/NX-Activity-Log#18]
I have a few forwarders in my Switch and I'd like to get their activity log too, is it possible?

**Comment by tallbl0nde** [comment]
It is somewhat possible, however the NSPs must have been created in a way that requires a user to be selected (which as far as I know hasn't been done yet). I _could_ still show the stats but it would likely be unreliable/incorrect. I'll definitely keep an eye on this and include it when it's possible :)

**Comment by furanzoni** [comment]
Sweet! Thanks, looking forward to it and great work.

**Comment by Stuntaneous** [comment]
Many of us only have the one user so it would still be useful as you describe.

**Comment by reinzanini** [comment]
I think the best way to implement would be, in some way (throwing random ideas now, but what about noting down the NSP's TitleID and then adding it?), to add the forwarder manually on your NX-Activity-Log. Otherwise you might get Tinfoil and shit between your games and personally I'd hate it.

**Comment by tallbl0nde** [comment]
I was actually thinking of something along those lines... I'd probably just search through the play log and any title IDs that aren't already shown can then be added.

**Comment by Aliasim800** [comment]
Yeah I actually tried making a forwarder but it didn't work. Whenever I open it It says "The Software Was Closed Because a Error Occured". So it would be great to have some forwarder support please. Prefferably allow the user selection screen for people to make their own instead of just uploading a pre-made forwarder in the next release. Thank You!

**Comment by tallbl0nde** [comment]
I'm not quite sure what you mean, I was never going to include another forwarder aside from the User Page one?

**Comment by jordanbelinsky** [comment]
Any update on this issue? Have a couple RetroArch rom forwarders that I'd love to see tracked!
